ansible_base/resource_registry/tasks/sync.py

```diff
@@ -317,8 +316,9 @@ def _cleanup_orphans(self, resource_type, manifest_list):
         if self.deleted_count:
             self.write(f"Deleting {self.deleted_count} orphaned resources")
             for orphan in resources_to_cleanup:
-                # If it was created in the latest X seconds, ignore it.
-                if orphan.content_object.created >= timezone.now() - timedelta(seconds=self.retain_seconds):
+                # Skip items that owned by this service, but are partially migrated
+                if orphan.service_id == service_id() and orphan.is_partially_migrated == True:
+                    # TODO: how do I log skip?
```

Comment:
If you want to log to the application log, I'd imagine you `import logging` and create a logger for this file.

If you mean write to stderr, you'd need to do something similar to how we track stdout, defining a `stderr` field for the `SyncExecutor` dataclass, and update the `write` method.